app/main.py

```diff
@@ -1,6 +1,14 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
```

Comment:
use `@wraps` decorator imported from functools. Read what the purpose of this decorator in inner function